corehq/apps/hqwebapp/forms.py

```diff
@@ -58,12 +58,12 @@ def clean(self):
 
         cleaned_data = super(EmailAuthenticationForm, self).clean()
         if not user:
-            consumer_user = ConsumerUser.objects.get_or_none(user=self.user_cache)
-            if consumer_user:
+            try:
+                _ = ConsumerUser.objects.get(user=self.user_cache)
```

Comment:
You can't assign to `_` as you are using that to mean `ugettext_lazy`, so that is getting overwritten here. If you aren't using this, you don't need to assign it to a variable.

Reply:
Fixed this